api/src/main/java/org/pmiops/workbench/cohortreview/ReviewTabQueryBuilder.java

```diff
@@ -13,92 +12,160 @@
-        "     source_value as sourceValue,\n" +
-        "     source_vocabulary as sourceVocabulary,\n" +
-        "     source_name as sourceName,\n" +
-        "     age_at_event as ageAtEvent\n" +
-        "     %s\n" +
-        "from `${projectId}.${dataSetId}.%s`\n";
-
-    private static final String WHERE_TEMPLATE =
-        "where person_id = @" + NAMED_PARTICIPANTID_PARAM + "\n" +
-        "order by %s %s, data_id\n" +
-        "limit %d offset %d\n";;
-
-    private static final String COUNT_TEMPLATE =
-      "select count(*) as count\n" +
-        "from `${projectId}.${dataSetId}.%s`\n" +
-        "where person_id = @" + NAMED_PARTICIPANTID_PARAM + "\n";
-
-    private static final String DETAILS_WHERE =
-      "where data_id = @" + NAMED_DATAID_PARAM + "\n";
-
-
-    public QueryJobConfiguration buildQuery(Long participantId,
-                                            DomainType domain,
-                                            PageRequest pageRequest) {
-        String tableName = DomainType.MASTER.equals(domain)
-          ? MASTER_TABLE : TABLE_PREFIX + domain.toString().toLowerCase();
-        String extraColumns = EXTRA_COLUMNS.get(domain.toString()) == null
-          ? "" : EXTRA_COLUMNS.get(domain.toString());
-        String finalSql = String.format(SQL_TEMPLATE + WHERE_TEMPLATE,
-          extraColumns,
-          tableName,
-          pageRequest.getSortColumn(),
-          pageRequest.getSortOrder().toString(),
-          pageRequest.getPageSize(),
-          pageRequest.getPage() * pageRequest.getPageSize());
-
-        Map<String, QueryParameterValue> params = new HashMap<>();
-        params.put(NAMED_PARTICIPANTID_PARAM, QueryParameterValue.int64(participantId));
-        return QueryJobConfiguration
-          .newBuilder(finalSql)
-          .setNamedParameters(params)
-          .setUseLegacySql(false)
-          .build();
-    }
+  private static final String NAMED_PARTICIPANTID_PARAM = "participantId";
+  private static final String TABLE_PREFIX = "p_";
+  private static final String MASTER_TABLE = "p_all_events";
 
-    public QueryJobConfiguration buildCountQuery(Long participantId,
-                                                 DomainType domain) {
-        String tableName = DomainType.MASTER.equals(domain)
-          ? MASTER_TABLE : TABLE_PREFIX + domain.toString().toLowerCase();
-        String finalSql = String.format(COUNT_TEMPLATE, tableName);
-        Map<String, QueryParameterValue> params = new HashMap<>();
-        params.put(NAMED_PARTICIPANTID_PARAM, QueryParameterValue.int64(participantId));
-        return QueryJobConfiguration
-          .newBuilder(finalSql)
-          .setNamedParameters(params)
-          .setUseLegacySql(false)
-          .build();
-    }
+  private static final String BASE_SQL_TEMPLATE =
+    "select person_id as personId,\n" +
+      "data_id as dataId,\n" +
+      "start_datetime as startDate,\n";
+
+  private static final String ALL_EVENTS_SQL_TEMPLATE =
```

Comment:
Maybe refactor these sql templates later to reduce some duplication.

Reply:
Yep.